Branch-SDK/Branch-SDK/ContentDiscoverer.m

```diff
@@ -0,0 +1,287 @@
+//
+//  ContentDiscoverer.m
+//  Branch-TestBed
+//
+//  Created by Sojan P.R. on 8/17/16.
+//  Copyright © 2016 Branch Metrics. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "ContentDiscoverer.h"
+#import "ContentDiscoveryManifest.h"
+#import "ContentPathProperties.h"
+#import "BNCPreferenceHelper.h"
+#import "BranchConstants.h"
+#import <UIKit/UIKit.h>
+#import <CommonCrypto/CommonDigest.h>
+
+
+@implementation ContentDiscoverer
+
+static ContentDiscoverer *contentViewHandler;
+UIViewController *lastViewController;
```

Comment:
It's best to declare all of these non-static variables in the private interface like this example: https://github.com/BranchMetrics/ios-branch-deep-linking/blob/master/Branch-SDK/Branch-SDK/BNCLinkCache.m#L11